src/gt4sd/frameworks/granular/arg_parser/parser.py

```diff
@@ -94,3 +98,6 @@ def parse_arguments_from_config(conf_file: Optional[str] = None) -> argparse.Nam
     result_namespace = argparse.Namespace(**result)
 
     return result_namespace
+
+
+_sentencepiece
```

Comment:
typo? I would remove this

Reply:
It's not a typo. This is needed because flake8 complains otherwise. The tests are executed with the `--disable-noqa` flag, so I dont see any other way of circumvening flake8 from complaining. The current configuration only ignores `__init__` files, see here: https://github.com/GT4SD/gt4sd-core/blob/d127590e2a312bda6c55f39304b70462f342ef4a/.github/workflows/tests.yaml#L45

If we import sentencepiece in the file, we have to use it somehow, otherwise we get a F401

Reply:
As I wrote above, this will not work because we run flake8 with `--disable-noqa` so we cannot overrule F401 errors, unless in `init` files.

Reply:
I would honestly prefer to see _sentencepiece just below the import because it's a bit weird to see this there without comments.